docker-compose.build.yaml

```diff
@@ -50,3 +50,10 @@ services:
       context: airbyte-migration
       labels:
         io.airbyte.git-revision: ${GIT_REVISION}
+  airbyte-temporal:
```

Comment:
should this be `temporal` instead of `airbyte-temporal`? I'm only basing this question off of the other entries in this file

Reply:
you're right, Done.